sdks/python/apache_beam/io/iobase.py

```diff
@@ -1180,15 +1180,47 @@ def check_done(self):
     raise NotImplementedError
 
   def try_split(self, fraction_of_remainder):
+    """Splits current restriction based on fraction_of_remainder.
+
+    Invoked when SDK receiving ProcessBundleSplitRequest during processing
+    bundle.
+
+    Args:
+      fraction_of_remainder: a fraction of (cur_pos, stop_pos).
```

Comment:
fraction_of_remainder: A hint as to the fraction of work the primary restriction should represent based upon the current known remaining amount of work.